docs/source/getting-started/installing.adoc

```diff
@@ -16,8 +16,8 @@ The following section describes how to install Minishift and the required depend
 == Prerequisites
 
 Minishift requires a hypervisor to start the virtual machine on which the OpenShift cluster
-is provisioned. Make sure that the hypervisor of your choice is installed and enabled on
-your system before you start Minishift.
+is provisioned. Ensure you have Virtualization Technology (VT-x) enabled in your system and
+the hypervisor of your choice is installed and working before you start Minishift.
```

Comment:
and > and that

change "working" to "enabled"/"running" (working is too ambiguous)